kinode.wit

```diff
@@ -1,13 +1,15 @@
-package kinode:process@0.8.0;
+package kinode:process@1.0.0;
 
 interface standard {
-    //
-    // System types:
-    //
+
+    /*      ˗ˏˋ ♡ ˎˊ˗     */
+    /*    System Types    */
```

Comment:
Given comment below (change `//` -> `///`), I'd prefer these `/*`s be `//`s. I think you changed them in order to have a different section header from the docs comments, but the `//` vs `///` will differentiate and have the benefit of not looking bad on non-monospaced viewers (like Github; check the diff here or View file)

Reply:
![image](https://github.com/user-attachments/assets/2b843d4e-97f8-4d3a-8779-4bcc188408fa)

Reply:
Oh good catch. Will swap in `//`